ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -103,7 +103,7 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
         // @ts-ignore
         let result = res?.getPerformanceProfiles;
         updateProgress({ showProgress : false });
-        if (typeof result !== "undefined") {
+        if (typeof result !== "undefined" && result.profiles.length !== 0) {
```

Comment:
What about new users, who have zero results? Are they not allowed to establish a subscription?

Reply:
Sorry, I didn't get this point. New user will not have any profiles, so they will be seeing this page. Is there something I am missing? 😓 
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/51183483/148887824-ea7c85f1-5dfa-4973-a69b-5462325513b3.png">

Reply:
So long as new users see this message, no we're good.